app/gui/src/model/execution_context/synchronized.rs

```diff
@@ -76,6 +76,8 @@ impl ExecutionContext {
             let this = Self { id, model, language_server, logger };
             this.push_root_frame().await?;
             info!(this.logger, "Pushed root frame.");
+            this.load_component_groups().await?;
+            info!(this.logger, "Loaded the component groups.");
```

Comment:
If you are using "the" it means that you are referring to a specific set of components groups, but I think you are not refering to a specific set here, just some component groups that you are getting from LS, right?